sky/shell/platform/mojo/BUILD.gn

```diff
@@ -44,4 +49,11 @@ mojo_native_application("mojo") {
     "//third_party/icu",
     "//url",
   ]
+
+  if (is_android) {
+    ldflags = [
+      "-lGLESv2",
```

Comment:
This is needed on Android or else Skia doesn't link because GrGLCreateNativeInterface_android.cpp references these symbols.